templates/test_app.py

```
@@ -0,0 +1,22 @@
+import pytest
+from app import app  # assumes app.py defines `app = Flask(__name__)`
+
+@pytest.fixture
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)